log/logger.go

```diff
@@ -11,10 +11,12 @@ import (
 	"github.com/allegro/akubra/log/sql"
 )
 
-const (
-	// ContextreqIDKey is Request Context Value key for debug logging
-	ContextreqIDKey = "reqID"
-)
+// ContextKey for logging context information
+type ContextKey string
+
+func (c ContextKey) String() string {
```

Comment:
What was wrong with old construction. Why you're implementing Stringer for named string type? And then not use this metrod?